client/src/components/User/index.js

```diff
@@ -1,26 +1,29 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import { Link } from 'react-router-dom';
 
-const User = (props) => {
-    const user = props.users.login;
-    const url = `/user/edit-profile/${user.id}`;
+import UserAvatar from './../UserAvatar';
 
-    return (
-        <div className="user_container">
-            <div className="avatar">
-                <img alt="avatar" src="/images/avatar.png"/>
-            </div>
-            <div className="nfo">
-                <div><span>Name:</span> {user.name}</div>
-                <div><span>Lastname:</span> {user.lastname}</div>
-                <div><span>Email:</span> {user.email}</div>
-                <div><span>Password:</span> * * * * * * </div>
-            </div>
-            <div className="text-center">
-                <Link to={url} className="button-link">Edit Profile</Link>
+import './styles.css';
+
+class User extends PureComponent {
```

Comment:
Fuck, I changed it to be a class-based component when I was trying different approaches with the functionalities and I forgot to change it back when I moved all the functionality to the UserAvatar component...

This should be just a functional component as it does not have any logic, I have to change it...